.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.5"
```

Comment:
Pythoscope doesn't support Python 3, so I don't think there's a point in listing python 3.5 in travis configuration.